lib/versioning/pep440/index.spec.ts

```diff
@@ -148,6 +148,9 @@ describe('versioning/pep440/index', () => {
     ${'!=1.2.3'}          | ${'bump'}     | ${'1.0.0'}     | ${'1.2.3'} | ${null}
     ${'!=1.2.3'}          | ${'replace'}  | ${'1.0.0'}     | ${'1.2.3'} | ${null}
     ${'!=1.2.3'}          | ${'pin'}      | ${'1.0.0'}     | ${'1.2.3'} | ${'==1.2.3'}
+    ${'==3.2.*,>=3.2.2'}  | ${'replace'}  | ${'3.2.2'}     | ${'4.1.1'} | ${'==4.1.*'}
+    ${'==3.2.*,>=3.2.2'}  | ${'replace'}  | ${'3.2.2'}     | ${'4.0.0'} | ${'==4.0.*'}
+    ${'>=1.0.0,<1.1.0'}   | ${'replace'}  | ${'1.0.0'}     | ${'1.2.0'} | ${'>=1.0.0,<1.3.0'}
```

Comment:
```suggestion
    ${'>=1.0.0,<1.1.0'}   | ${'replace'}  | ${'1.0.0'}     | ${'1.2.0'} | ${'>=1.2.0,<1.3.0'}
```

Keep same width as before

Reply:
Range strategy = replace should keep the same range width as before (e.g. minor or major range) and not widen the range